src/SUMMARY.md

```diff
@@ -1,11 +1,13 @@
 # Rust Forge
 
 [Overview](./README.md)
-- [Chat Platforms](./chat/README.md)
+- [Platforms](./chat/README.md)
+  - [GitHub](./github.md)
```

Comment:
Minor nit: Could this list be sorted alphabetically?

Reply:
I had put it at the top because I thought it was the most prominent platform tool rust-lang has :smile: but ok it makes sense to sort them alphabetically :+1: